aiidalab_widgets_base/computational_resources.py

```diff
@@ -835,14 +835,13 @@ def _configure_computer(self, computer: orm.Computer):
             "use_login_shell": self.use_login_shell.value,
             "safe_interval": self.safe_interval.value,
         }
-        if "user" in sshcfg:
```

Comment:
@unkcpz I suspect this change broke the setup for local computer, reported in #417.
Could you please revert it and and a test for localhost computers?
(not a huge priority for me)

Reply:
Ha! You are correct, that makes sense, I'll reconsider how this part of the code should work. It currently leads to two new small issues.